telcell/utils/utils.py

```diff
@@ -0,0 +1,6 @@
+from telcell.data.models import Track
```

Comment:
File name makes this module `telcell.utils.utils`, seems a bit double.

Would this function make sense in the same module as `Track` (`telcell.data`)?

Reply:
Moved.